### ISSUE TITLE
Gulp watch doesnt update scss or js

### ISSUE THREAD

**Issue by zkhan96** [RyanFitzgerald/devportfolio#81]
The path for the watch script in the gulpfile.js is wrong:

```
gulp.task('watch', function() {
    gulp.watch('js/scripts.js', gulp.series('scripts'));
    gulp.watch('scss/styles.scss', gulp.series('styles'));
});
```

Should be:
```
gulp.task('watch', function() {
    gulp.watch('./js/scripts.js', gulp.series('scripts'));
    gulp.watch('./scss/styles.scss', gulp.series('styles'));
});
```

**Comment by NikulGoyani369** [comment]
![image](https://user-images.githubusercontent.com/19364203/77529210-79bc8c80-6e8f-11ea-8f59-03ffb413a7ad.png)

@zkhan96  do you now all step how to run this project after clone in my laptop??

**Comment by RyanFitzgerald** [comment]
Resolved in `1.2.2`
